Shapes.java

```diff
@@ -1,25 +1,52 @@
-import java.util.*;
+import java.util.Scanner;
+import java.lang.Math;
 
-public class Shapes{
+class Shapes{
   public static void main(String[] args) {
     Scanner input = new Scanner(System.in);
-    System.out.print("Enter the side length: ");
+
+    System.out.print("What is the side length for squares? ");
     double side = input.nextDouble();
+
     square(side);
-  }
 
-  /*
-   * N: square
-   * P: to calculate the area of any square
-   * I: a side measure (double)
-   * R:
-   */
-  public static void square(double s){
-    //side times side
-    double a = s * s;
+    System.out.print("What is the base for triangles? ");
+    double base = input.nextDouble();
+    System.out.print("What is the height for triangles? ");
+    double height = input.nextDouble();
+
+    triangles(base, height);
+
+    System.out.print("What is side 1 triangles? ");
+    double side1 = input.nextDouble();
+    System.out.print("What is side 2 triangles? ");
+    double side2 = input.nextDouble();
+    System.out.print("What is side 3 triangles? ");
+    double side3 = input.nextDouble();
+
+    trianglesSides(side1, side2, side3);
 
-    //generate some output
-    System.out.println("A square with side " + s + " has an area of " + a);
-  }//close square method
+    System.out.print("What is the radius for the circle? ");
+    double radius = input.nextDouble();
 
+    circle(radius);
+  }
+  public static void square(double s){
```

Comment:
Nice job using `static` methods here! Can you think of a way to have a `return` type that isn't `void`?